codebase/superdarn/src.bin/tk/tool/fit_median_filter.1.0/fit_median_filter.c

```diff
@@ -0,0 +1,322 @@
+/* fit_median_filter.c
+   ==========
+   
+Removes salt & pepper noise from a fitacf file using median filtering. 
+Filtering is performed separately for each beam and channel. 
+Output is a fitacf file.
+
+(C) Copyright 2021 E.C.Bland
```

Comment:
Just double checking U of S or Unis is not needed on this line? Not sure how your U of S contract worked out.